src/Tools/dotnet-user-secrets/test/SecretManagerTests.cs

```diff
@@ -299,7 +299,7 @@ public void Clear_Secrets(bool fromCurrentDirectory)
         foreach (var keyValue in secrets)
         {
             Assert.Contains(
-                string.Format(CultureInfo.InvariantCulture, "Successfully saved {0} = {1} to the secret store.", keyValue.Key, keyValue.Value),
+                string.Format(CultureInfo.InvariantCulture, "Successfully saved {0} to the secret store.", keyValue.Key, keyValue.Value),
```

Comment:
Should `keyValue.Value` be taken out of this now that there isn't a placeholder for it?

Reply:
```suggestion
                string.Format(CultureInfo.InvariantCulture, "Successfully saved {0} to the secret store.", keyValue.Key),
```